### DISCUSSION TITLE
Infinite [ClientTaskWorker - INFO - Finished one task run for client: siteNumber]

### DISCUSSION THREAD

**Discussion by pygabc1** [NVIDIA/NVFlare#1396] [Q&A] [answered]
Sometimes, the following logo is printing "infinitely" without finishing or stopping.

> ClientTaskWorker - INFO - Finished one task run for client: siteNumber

Any advice would be appreciated. It is linux OS and 3.8 python, and 2.2 NVFlare version.

**Comment by YuanTingHsieh** [comment, marked as answer]
Hi @pygabc1 , if you are using simulator, this message will keep printing until the job is finished.
All the examples should run to completion given enough resources in your machine and correct installation.

To verify your installation is correct, please run the hello-numpy-sag example and see if it completes.
